src/ethereum/info/etcInfo.ts

```diff
@@ -5,7 +5,16 @@ import { makeMetaTokens } from '../../common/tokenHelpers'
 import type { EthereumTools } from '../ethPlugin'
 import type { EthereumFees, EthereumSettings } from '../ethTypes'
 
-const builtinTokens: EdgeTokenMap = {}
+const builtinTokens: EdgeTokenMap = {
+  '2c78f1b70ccf63cdee49f9233e9faa99d43aa07e': {
+    currencyCode: 'DAI',
+    displayName: 'Dai Stablecoin',
+    denominations: [{ name: 'DAI', multiplier: '1000000000000000000' }],
+    networkLocation: {
+      contractAddress: '0x2c78f1b70ccf63cdee49f9233e9faa99d43aa07e'
```

Comment:
Super low tx volume (1 tx this month) maybe we use WETC https://explorer.bitquery.io/ethclassic/token/0x82a618305706b14e7bcf2592d4b9324a366b6dad or another one https://explorer.bitquery.io/ethclassic/tokens

Reply:
Fine because we want to have the token already